2206 ootbc kafka rabbitmq document intermediate catch event (#2208)
camunda/camunda-platform-docs: docs/components/connectors/out-of-the-box-connectors/rabbitmq-inbound.md

## Prerequisites

To use the **RabbitMQ Connector**, you need to have installed a RabbitMQ server and create the relevant [credentials](https://www.rabbitmq.com/passwords.html).

> It is highly recommended to use Camunda secrets to store credentials, so you don't expose sensitive information directly from the process. See [this appendix entry](#how-do-i-store-secrets-for-my-connector) to learn more.

Reviewer: ```suggestion
Using Camunda secrets to store credentials is highly recommended, so you do not expose sensitive information directly from the process. See [this appendix entry](#how-do-i-store-secrets-for-my-connector) to learn more.
```
A little rewording and removing the contraction (don't -> do not).

Author: Done

The **Activation** section allows you to configure the custom activation conditions for the RabbitMQ Consumer Connector.

#### Correlation key

![RabbitMQ Intermediate Catch Event](../img/connectors-rabbitmq-inbound-intermediate.png)

> The correlation key fields are only applicable for the Intermediate Event **RabbitMQ Connector**.

Reviewer: ```suggestion
The correlation key fields are only applicable for the intermediate event **RabbitMQ Connector**.
```

Author: Done
